liquibase_postgresql.js

```diff
@@ -0,0 +1,87 @@
+const liquibase = require('node-liquibase');
```

Comment:
same, let's move both of these examples into `./examples/` to make node users feel more comfortable.

Reply:
Done.